core/src/test/java/com/mapzen/android/graphics/MapzenMapTest.java

```diff
@@ -293,10 +293,10 @@
     verify(overlayManager).onFling(1f, 2f, 3f, 4f);
   }
 
-  @Test public void setRotateResponder_shouldInvokeMapController() {
+  @Test public void setRotateResponder_shouldNotInvokeMapController() {
```

Comment:
I agree once these tests are added we are good to merge. Tests should now automatically run on Circle CI when the next commit is made.